barcode/upc.py

```diff
@@ -8,21 +8,17 @@
 
 from barcode.base import Barcode
 from barcode.charsets import upc as _upc
-from barcode.errors import (
-    IllegalCharacterError,
-    NumberOfDigitsError,
-)
+from barcode.errors import IllegalCharacterError, NumberOfDigitsError
 
 
 class UniversalProductCodeA(Barcode):
     """Initializes new UPC-A barcode.
 
-    :parameters:
-        upc : String
-            The upc number as string.
-        writer : barcode.writer Instance
-            The writer to render the barcode (default: SVGWriter).
-        make_ean: boolean
+    :param str upc: The upc number as string.
```

Comment:
Hi there! Thanks for correcting this. Docs in general need some love, so I appreciate you addressing some of it.

These parameters, however, apply to `__init__`, would you mind moving it a to line 33? This was a preexisting issue really, but it's probably a good chance to address that.

Again, thanks for contributing these fixes. :relieved:

Reply:
Fixed with 36bc0bb